packages/helpermodules/measurement_logging/write_log.py

```diff
@@ -209,13 +209,14 @@ def create_entry(log_type: LogType, sh_log_data: LegacySmartHomeLogData, previou
             log.exception("Fehler im Werte-Logging-Modul für EV "+str(ev))
 
     counter_dict = {}
-    for counter in data.data.counter_data:
+    for counter in data.data.counter_data.values():
         try:
-            if "counter" in counter:
+            if ("counter" in counter and
+                    counter.num != data.data.counter_all_data.data.config.home_consumption_source_id):
```

Comment:
Was spricht dagegen, die Daten des Zählers auch zu speichern?

Reply:
Das hatte ich mir bei unserem Gespräch notiert, dass in den Diagramm-Daten der Zähler nur als HV erfasst werden soll, damit man nicht zwei Datenreihen mit den gleichen Werten hat.